xml/System.Net.Http/HttpResponseMessage.xml

```diff
@@ -484,9 +484,21 @@
         <ReturnType>System.Net.Http.Headers.HttpResponseHeaders</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the collection of trailing headers included in an HTTP response.</summary>
+        <value>The collection of trailing headers in the HTTP response.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+Forbidden headers will be ignored.
+
+Since trailing headers are send as an HTTP header appended at the end of the response message, the `TrailingHeaders` property returns an empty <xref:System.Net.Http.Headers.HttpResponseHeaders> instance if it is accessed and the response content has not been read completely. 
```

Comment:
```suggestion
Since trailing headers are sent as an HTTP header appended at the end of the response message, the `TrailingHeaders` property returns an empty <xref:System.Net.Http.Headers.HttpResponseHeaders> instance if it is accessed and the response content has not been read completely. 
```